ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/AutoSizeText.kt

```diff
@@ -0,0 +1,66 @@
+package com.revenuecat.purchases.ui.revenuecatui.helpers
+
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithContent
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.isUnspecified
+
+@SuppressWarnings("LongParameterList")
+@Composable
+fun AutoResizedText(
```

Comment:
😬 Compose doesn't provide this? Yikes.
We should probably avoid using this for the most part because it's not accessibility friendly at all. This is the only place where I'm using it in iOS.

For now this solution is fine I think 👍🏻  

A couple of suggested changes:
- Move this to `composables`
- Add a docstring to explain what this does